### ISSUE TITLE
Support of ruby 2.2.7

### ISSUE THREAD

**Issue by ShockwaveNN** [travis-ci/travis-ci#7535]
Ruby 2.2.7 released yesterday
So I think it need to be added to build matrix like this was done with Ruby 2.4.1 some time ago (https://github.com/travis-ci/travis-ci/issues/7512)
Currently using it cause error:
```
rvm use 2.2.7 --install --binary --fuzzy
ruby-2.2.7 is not installed - installing.
Searching for binary rubies, this might take some time.
Requested binary installation but no rubies are available to download, consider skipping --binary flag.
Gemset '' does not exist, 'rvm ruby-2.2.7 do rvm gemset create ' first, or append '--create'.
The command "rvm use 2.2.7 --install --binary --fuzzy" failed and exited with 2 during .
```

**Comment by timbru31** [comment]
@ShockwaveNN ruby 2.2.7 is working fine for me now 👍 
@BanzaiMan could you kick the build matrix for newly [released 2.3.4](https://www.ruby-lang.org/en/news/2017/03/30/ruby-2-3-4-released/) version?

**Comment by tagliala** [comment]
Hi,

I'm the author of #7512 

2.2.7 is working for me, 2.3.4 isn't.

Should I open another issue for 2.3.4 ?

**Comment by BanzaiMan** [comment]
[2.2.7](https://travis-ci.org/travis-ci/travis-rubies/builds/216311399) is available for Linux, OS X 10.11 and macOS 10.12.

[2.3.4](https://travis-ci.org/travis-ci/travis-rubies/builds/216848482) for Linux.

**Comment by ShockwaveNN** [comment]
Everything seems fine with ruby 2.2.7 and 2.3.4 now
